ciao-cli/main.go

```diff
@@ -139,8 +141,11 @@ const (
 	ciaoPasswordEnv    = "CIAO_PASSWORD"
 	ciaoComputePortEnv = "CIAO_COMPUTEPORT"
 	ciaoTenantNameEnv  = "CIAO_TENANT_NAME"
+	ciaoCACertFileEnv  = "CIAO_CA_CERT_FILE"
```

Comment:
@rbradford Could you please document this new env variable in the ciao-cli README.md file ?
The code itself looks good to me.